notebooks/examples/Identify Inspected Buildings.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Detect Inspected Buildings\n",
+    "\n",
+    "In damage assessment inspections, it's common that the structure needs to be fully inspected. One way to ensure that, is by having inspectors walk around the structure. In this example notebook, we'll how tracks can be used accomplish to identify structures that have been inspected. We'll use some sample data in Portland, ME to demonstrate this."
```

Comment:
Bad comma - either: `One way to ensure that is by having inspectors walk around the structure` or `One way to ensure that: having inspectors walk around the structure`

Reply:
Missing "see" in between "We'll" and "how"

Reply:
Remove "accomplish"